### PR TITLE
rename default_wait_time to default_max_wait_time - Issue #1442

### PR DESCRIPTION
**Pull request by twalpole**
Implementation of the request/discussion in Issue #1442

### CODE REVIEW

File: lib/capybara.rb [hunk 1]
```diff
@@ -295,6 +295,18 @@ def HTML(html)
...
       end
     end
+    
+    # @deprecated Use default_max_wait_time instead
+    def default_wait_time
+      deprecate('default_wait_time', 'default_max_wait_time')
+      default_max_wait_time
+    end
+    
+    # @deprecated USe default_max_wait_time= instead
```
Comment (abotalov) [inline lib/capybara.rb]: "USe" -> "Use" :)

File: lib/capybara.rb [hunk 2]
```diff
@@ -44,7 +44,7 @@ class << self
     # [raise_server_errors = Boolean]     Should errors raised in the server be raised in the tests? (Default: true)  
     # [server_errors = Array\<Class\>]    Error classes that should be raised in the tests if they are raised in the server and Capybara.raise_server_errors is true (Default: [StandardError])  
     # [default_selector = :css/:xpath]    Methods which take a selector use the given type by default (Default: :css)  
-    # [default_wait_time = Integer]       The maximumnumber of seconds to wait for asynchronous processes to finish (Default: 2)  
+    # [default_max_wait_time = Integer]   The maximum number of seconds to wait for asynchronous processes to finish (Default: 2)
```
Comment (abotalov) [inline lib/capybara.rb]: Actually, the type is not Integer but Numeric, I think

File: lib/capybara/node/document_matchers.rb [hunk 3]
```diff
@@ -9,7 +9,7 @@ module DocumentMatchers
       #     @param string [String]           The string that title should include
       #   @overload $0(regexp, options = {})
       #     @param regexp [Regexp]           The regexp that title should match to
-      #   @option options [Numeric] :wait (Capybara.default_wait_time) Time that Capybara will wait for title to eq/match given string/regexp argument
+      #   @option options [Numeric] :wait (Capybara.default_max_wait_time) Time that Capybara will wait for title to eq/match given string/regexp argument
```
Comment (abotalov) [inline lib/capybara/node/document_matchers.rb]: "Time" -> "Maximum time"

**Comment by twalpole** [comment]
@abotalov thanks for the review - Fix for those comments coming shortly

**Pull request merged**